dockstore-webservice/src/main/java/io/dockstore/webservice/resources/EntryResource.java

```diff
@@ -743,7 +783,7 @@ public boolean canExamine(User user, Entry entry) {
 
     @Override
     public boolean canWrite(User user, Entry entry) {
-        return AuthenticatedResourceInterface.super.canWrite(user, entry) || AuthenticatedResourceInterface.canDoAction(permissionsInterface, user, entry, Role.Action.WRITE);
+        return isWritable(entry) && (AuthenticatedResourceInterface.super.canWrite(user, entry) || AuthenticatedResourceInterface.canDoAction(permissionsInterface, user, entry, Role.Action.WRITE));
```

Comment:
Interesting quirk if I understand correctly (please correct if I do not)
This flag means that most actions directly initiated by users will be blocked but github app updates should remain (which seems ok). 

Will this also block curators/admins from labeling? (Probably also ok)

Reply:
Yup, the intention is that if an entry is archived, it is [mostly] completely read only.  All actions that would change it, except to unarchive it, are disallowed.  To that end, this PR adds logic in the push code that will make pushes to archived entries fail (and log something reasonable).

Reply:
Ah, I see. It's the change in `createOrGetWorkflow` that I missed that affects github releases in particular